about.html

```diff
@@ -0,0 +1,109 @@
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a href="" id="Campaign">CC Campaign</a></li>
+            </ul>
+        </nav>
+        <div class="meuncomponents">
+            <button class="menubtn" type="submit"><img src="image/hamburger.JPG" id ="menu"></button>
+            <div class="menuspace"></div>
+        </div>
+        <div class="popup">
+            <a href=""> Home </a>
+        </div>
+        <div class="header-text">
+            <h2>"Hello! Sharing World"</h2>
+                <h1>National Reneweable Energy Submit 2023</h1>
+           </div>
+           <div class="header-paragraph">
+            <p>This is a yearly event where over 300 experts and professionals in the field of renewable energy technology converge to discuss the latest trend in the field. Outstanding professionals are also recognized during this period. The submit is supported by the United Nations and the IREA</p>
+           </div>
+
+           <div class="header-footer">
+            <p class="message">For more enquiry on participating in the submit, please reach out to Omotayo</p>
+            <p class="email"> akandeabdul@gmail.com</p>
+           </div>
+
+        </section>
+        
+    <section class="main-program">
+            <div class="main-heading">
+                <h3>Renewable Energy Submit 2023</h3>
+                <hr>
+                <p>The decision to host the submit in Lagos, Nigeria was decided in an election at Paris in November, 2022. The logo of the submit was release in january, 2023</p>
+            </div>
+            <div class="logo">
+
+            </div>
+            <hr class="divider">
+            <div class="gallery">
+                <h3>See Images of Past Conferences</h3>
+                <p>Take a look at the past two (2) submits that took place in Atlanta and London</p>
+            </div>
+            <div class="galleryimage">
+            <div class="Atlanta">
+                <div class="textdisplay">
+                <h3> Atlanta</h3>
+                <p>Renewable Energy Submit in Atlanta 2020</p>
+            </div>
+            </div>
+            <div class="London">
+                <div class="textdisplay">
+                <h3> London</h3>
+                <p>Renewable Energy Submit in London 2021</p>
+            </div>
+        </div>
+            </div>
+    </section>
+    <section class="footer">
+        <div class="partner">
+            <h3 >Partner</h3>
+            <hr>
+            <ul class="partner-list">
+                <li><img src="image/Shell_logo.svg.png" class="logopatner"></li>
+                <li><img src="image/african_union-emblem.png" class="logopatner"></li>
+                <li><img src="image/UN_emblem_blue.svg.png" class="logopatner"></li>
+                <li><img src="image/EU.jpg" class="logopatner"></li>
+                <li><img src="image/Renewable.avif" class="logopatner"></li>
```

Comment:
- [ ] You have done incredibly well by using img tag. However, you should ensure that the alt has a text. This is because alt text is critical for those with disabilities because it describes the function and appearance of a photo or graphic they cannot see. Kindly ensure to make your page more accessible by implementing the alt text 👍 .